content/en/real_user_monitoring/guide/session-replay-getting-started.md

```diff
@@ -0,0 +1,161 @@
+
+*Supported browsers*: The Session Replay recorder supports all the browsers supported by the RUM Browser SDK with the exception of IE11. See the [browser support table][3].
+
+#### Configuration
+
+The usual [RUM initialization parameters][4] are all supported. 
+
+In addition, the Session Replay recorder accepts a `manualSessionReplayRecordingStart` option. If set to `true`, the Session Replay recording will not start automatically when calling `init()`. To then start the recording, call `startSessionReplayRecording()`. This can be useful to conditionally start the recording (e.g, to only record authenticated user sessions):
+
+``` javascript
+DD_RUM.init({
+	manualSessionReplayRecordingStart: true
+})
+if (user.isAuthenticated) {
+	DD_RUM.startSessionReplayRecording()
+}
+```
+
+The Session Replay recording can be stopped by calling `stopSessionReplayRecording()`.
+
+### Sensitive & personal data obfuscation
+
+By default, any input events on HTML input elements of type `password`, `email` and `tel` are ignored.
+
+These events can be ignored on other elements as well, by either setting the data attribute `data-dd-privacy` to `input-ignored`, or adding a class of `dd-privacy-input-ignored`. For example, any inputs on the following form will be ignored:
+
+``` html
+<form method="post" data-dd-privacy=”input-ignored”>
+    <input type="text" name="name" id="name" required>
+    <input type="number" name="age" id="age" required>
+    <input type="email" name="email" id="email" required>
+    <input type="submit" value="submit">
+</form>
+```
+
+Additionally, HTML elements can be fully obfuscated. These elements are replaced with a white block at recording time, as seen in this example of the navbar of a replay of a Datadog session: 
+
+{{< img src="real_user_monitoring/guide/replay-hidden.png" alt="Replay Hidden Example">}}
+
+Obfuscate elements by setting the data attribute `data-dd-privacy` to hidden, or by adding a class of `dd-privacy-hidden`. Elements marked as hidden will *not* have their content recorded and the PII will not be sent to Datadog. For example, the following div will be obfuscated and replaced in the replay by a white block of the same size:
+
+``` html
+<div id=”profile-info” data-dd-privacy=”hidden”>
+    <p>Name: John Doe</p>
+    <p>Birth date: June 6th, 1987</p>
+</div>
+```
+
+## Troubleshooting
+
+### Some HTML elements are not visible at replay
+
+Session Replay does not support the following HTML elements at the moment: `iframe`, `video`, `audio`, `canvas`, as well as Web Components.
+
+### Fonts or images do not render properly
+
+A Session Replay is not video, but an actual iframe rebuilt based on snapshots of the DOM. The replay thus depends on the various assets of the page: fonts & images.
+
+Several reasons might explain why assets are not available at the time of the replay.
+
+1. The resource does not exist anymore. For example, it was part of a previous deployment.
```

Comment:
```suggestion
- The resource does not exist anymore. For example, it was part of a previous deployment.
```
changing this to a bulleted list because we only use an ordered list if you need to complete steps in the specific order